paddle/fluid/platform/cupti_cbid_str.h

```diff
@@ -0,0 +1,903 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+#include <string>
+#include <vector>
+
+#ifdef PADDLE_WITH_CUPTI
+
+namespace paddle {
+namespace platform {
+
+std::vector<std::string> runtime_cbid_str = {
+    /* NOTE(liangdun) : the code below was generated by bash commands, if the
+      cuda
+       version updated, those code may need to be re-generated:
+       grep CBID /usr/local/cuda/extras/CUPTI/include/cupti_runtime_cbid.h | \
+      awk '{print "\""$1"\","}'
```

Comment:
We should not record all the activation of `runtime`, because most of them is not used in Fluid.

Reply:
`cuptiActivityEnable(CUPTI_ACTIVITY_KIND_RUNTIME)` will trace all runtime api, I can not control it.